Index: xmlada-2013-src/Makefile.in
===================================================================
--- xmlada-2013-src.orig/Makefile.in
+++ xmlada-2013-src/Makefile.in
@@ -82,8 +82,8 @@ run_test: force
 run_test_valgrind: test
 	cd dom/test; valgrind --tool=memcheck --leak-check=full ./testxml -auto
 
-docs: force
-	-${MAKE} -e -C docs html latexpdf
+# docs: force
+# 	-${MAKE} -e -C docs html latexpdf
 
 # Rule for installing the modules
 %_inst: force
@@ -145,8 +145,8 @@ install_library_type: install_dirs ${MOD
 	${INSTALL} distrib/xmlada_schema.gpr         ${prefix}/lib/gnat/
 	${INSTALL} distrib/xmlada_unicode.gpr        ${prefix}/lib/gnat/
 	${INSTALL} xmlada-config  ${bindir}
-	-$(CP) -r docs/_build/html/* $(datadir)/doc/xmlada/ 2>/dev/null
-	-$(CP) -r docs/_build/latex/XMLAda.pdf $(datadir)/doc/xmlada/ 2>/dev/null
+# -$(CP) -r docs/_build/html/* $(datadir)/doc/xmlada/ 2>/dev/null
+# -$(CP) -r docs/_build/latex/XMLAda.pdf $(datadir)/doc/xmlada/ 2>/dev/null
 	${CHMOD} +x ${bindir}/xmlada-config
 	${INSTALL} distrib/xmlada_gps.py ${datadir}/gps/plug-ins/
 
@@ -168,7 +168,7 @@ clean_%: force
 	-gnat clean -XLIBRARY_TYPE=$* -q -Pschema/test/schema_test.gpr
 
 clean: clean_relocatable clean_static
-	cd docs; ${MAKE} -e clean
+#	cd docs; ${MAKE} -e clean
 
 distclean: clean
 	${RM} dom/test/default.gpr
